getting-started/installation/secure-bootstrap.markdown

```diff
@@ -41,7 +41,7 @@ settings. So after we have set-up our hub (using the standard procedure
 of `cf-agent --bootstrap $HUB_IP`) we take care of the following:
 
 * `cf-serverd` must never accept a connection from a client presenting an
-  untrusted key. [Disable automatic key trust][Masterfiles Policy Framework#trustkeysfrom]
+  untrusted key. [Disable automatic key trust][Masterfiles Policy Framework#Automatic bootstrap - Trusting keys from new hosts with trustkeysfrom]
```

Comment:
the link here didn't work out, the generator doesn't complain but the link doesn't work. Looks like the generated link has one fewer spaces turned into dashes. I will fix with another commit to master and we can cherry pick back into this PR.

as generated:
```
http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-build-and-deploy-docs-3.21-7/output/_site/reference-masterfiles-policy-framework.html#automatic-bootstrap--trusting-keys-from-new-hosts-with-trustkeysfrom
```

what actually works:
```
http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-build-and-deploy-docs-3.21-7/output/_site/reference-masterfiles-policy-framework.html#automatic-bootstrap---trusting-keys-from-new-hosts-with-trustkeysfrom
```